Profiles/eAfspraak-Appointment.xml

```diff
@@ -326,8 +334,8 @@
       <alias value="ContactOnderwerp" />
       <mapping>
         <identity value="art-decor-eafspraak" />
-        <map value="eafspraak-dataelement-4" />
-        <comment value="IdentificationNumber" />
+        <map value="eafspraak-dataelement-22" />
+        <comment value="ContactDescription" />

```

Comment:
This is not the correct mapping. ConctactDesciption is already present at Appointment.comment. 

I believe this should be ContactSubject - eafspraak-dataelement-21